backend/python/migrations/versions/dedc5f7be10c_initial_migration.py

```diff
@@ -36,7 +36,11 @@ def upgrade():
         sa.Column("first_name", sa.String(), nullable=False),
         sa.Column("last_name", sa.String(), nullable=False),
         sa.Column("auth_id", sa.String(), nullable=False),
-        sa.Column("role", sa.Enum("User", "Admin", name="roles"), nullable=True),
+        sa.Column(
```

Comment:
From what I remember, we shouldn't be editing an existing migration but instead creating a new migration where the name of the migration is the change that we made. So, we'd want to have a migration file where we're adding relief staff as a role. Outlined [here](https://uwblueprint.github.io/starter-code-v2/docs/migrations/python-migrations.html#how-to-create-and-run-a-migration)

I may be incorrect with my thinking here, so this is definitely up to discussion.

Reply:
you're right, a new migration file was created for the changes and the initial migration has been restored